app/src/main/java/ch/epfl/sweng/erpa/activities/GameListActivity.java

```diff
@@ -105,12 +105,26 @@ public boolean onOptionsItemSelected(MenuItem item) {
     public boolean onOptionItemViewSelected(int id) {
```

Comment:
Can you please rename the `id` variable to something more verbose? What kind of id is it?
E.g. If this called with the id of a menu element pressed on an action view, call it something like actionMenuElementId